test/specs.js

```diff
@@ -179,6 +194,22 @@ describe('module-alias', function () {
     expect(bar).to.equal('Hello from baz')
     expect(src).to.equal('Hello from foo')
   })
+
+  if (semver.gte(process.version, '8.9.0')) {
+    it('should support the options argument', function () {
```

Comment:
Please, put it into a `describe` section so that you can create the spy in `beforeEach` and restore it in `afterEach`. Right now the spy is applied to `_resolveFilename` and never restored

Reply:
Once you've required module-alias, spying on `Module._resolveFilename` won't work.